docs/reference/System_variable.md

```diff
@@ -104,6 +104,10 @@ SELECT /*+ SET_VAR(query_timeout = 1) */ sleep(3);
 
   Used to set the default storage format used by the storage engine of the computing node. The currently supported storage formats are `alpha` and `beta`.
 
+* default_table_compression
+
+  Set the default compression algorithm for table storage, supported compression algorithms are: `snappy, lz4, lz4_frame, zlib, zstd, gzip, deflate, bzip2, lzo, brotli`.
```

Comment:
AFAIK we don't support deflate, lzo, brotli

Reply:
fixed